### ISSUE TITLE
Failed to set up error

### ISSUE THREAD

**Issue by RohitRajendran** [RobertDWhite/hue_circadian_lighting_bridge#4]
Hi! I was trying to set up the component but I'm getting the following error when it tries to set up:

``` 
This error originated from a custom integration.

Logger: custom_components.circadian_lighting_bridge
Source: custom_components/circadian_lighting_bridge/__init__.py:271
Integration: Circadian Lighting Bridge (documentation, issues)

Failed to connect to the Philips Hue bridge at 192.168.4.48. API Key: XXX check the host and try again.
```

Any suggestions?

**Comment by RobertDWhite** [comment]
Is the Hue bridge set up in Home Assistant correctly, where you went through the steps to actually press the button on the bridge to link it to HA? Connection issues are likely due to something between your HA environment and the bridge itself, like separate VLANs or something.

**Comment by RohitRajendran** [comment]
> Is the Hue bridge set up in Home Assistant correctly, where you went through the steps to actually press the button on the bridge to link it to HA? Connection issues are likely due to something between your HA environment and the bridge itself, like separate VLANs or something.

Yup, I had gone through that method to set up the Hue bridge and I have no issues controlling the lights from Home Assistant and the Circadian Lighting plugin works with no problem. Does this connect to the Hue bridge differently than the Circadian Lighting plugin does?
